modules/doc/content/getting_started/installation/osx_other.md

```diff
@@ -0,0 +1,32 @@
+# Mac OS High Sierra or Older
+
+!include getting_started/minimum_requirements.md
+
+## Prerequisites
+
+The following, is required to be installed before you can begin using MOOSE.
+
+Due to the unfortunate difficulty Apple has placed on its users wishing to develop on platforms older than Mojave, it is strongly encouraged users upgrade to Mojave.
```

Comment:
Ha.

Reply:
Downloading the mac OS Catalina Beta to stay ahead of the curve!

Reply:
Wonder if they will ever remove the need to install the headers package